tests/expectations/tests/derive-hash-template-inst-float.rs

```diff
@@ -102,31 +102,13 @@ fn __bindgen_test_layout_foo_open0_int_close0_instantiation() {
         4usize,
         concat!(
             "Size of template specialization: ",
-            stringify!(foo<::std::os::raw::c_int>)
-        )
-    );
-    assert_eq!(
-        ::std::mem::align_of::<foo<::std::os::raw::c_int>>(),
-        4usize,
-        concat!(
-            "Alignment of template specialization: ",
-            stringify!(foo<::std::os::raw::c_int>)
+            stringify ! ( foo < :: std :: os :: raw :: c_int > )
         )
     );
+    assert_eq ! ( :: std :: mem :: align_of :: < foo < :: std :: os :: raw :: c_int > > ( ) , 4usize , concat ! ( "Alignment of template specialization: " , stringify ! ( foo < :: std :: os :: raw :: c_int > ) ) );
 }
 #[test]
 fn __bindgen_test_layout_foo_open0_float_close0_instantiation() {
-    assert_eq!(
-        ::std::mem::size_of::<foo<f32>>(),
-        4usize,
-        concat!("Size of template specialization: ", stringify!(foo<f32>))
-    );
-    assert_eq!(
-        ::std::mem::align_of::<foo<f32>>(),
-        4usize,
-        concat!(
-            "Alignment of template specialization: ",
-            stringify!(foo<f32>)
-        )
-    );
+    assert_eq ! ( :: std :: mem :: size_of :: < foo < f32 > > ( ) , 4usize , concat ! ( "Size of template specialization: " , stringify ! ( foo < f32 > ) ) );
+    assert_eq ! ( :: std :: mem :: align_of :: < foo < f32 > > ( ) , 4usize , concat ! ( "Alignment of template specialization: " , stringify ! ( foo < f32 > ) ) );
```

Comment:
I think if you update to the latest `rustfmt` then it should preserve the old formatting here.

```
$ rustup update nightly
$ cargo +nightly install rustfmt-nightly
```